library/general/src/lib/config_file/augeas_parser.rb

```diff
@@ -0,0 +1,7 @@
+require "augeas"
+
+module ConfigAgent
```

Comment:
yeah, typo, original name and then I recognize I do not like agent term, when it is in fact configuration file.